context/app/static/js/components/Detail/DerivedDatasetsSection/DerivedDatasetsSection.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import DerivedEntitiesSectionWrapper from 'js/components/Detail/DerivedEntitiesSectionWrapper';
+import DerivedEntitiesTable from 'js/components/Detail/DerivedEntitiesTable';
+import DerivedEntitiesSectionHeader from 'js/components/Detail/DerivedEntitiesSectionHeader';
+
+function DerivedDatasetsSection({ datasets, uuid, isLoading, sectionId }) {
+  const entityType = 'Dataset';
+  return (
+    <DerivedEntitiesSectionWrapper
+      isLoading={isLoading}
+      sectionId={sectionId}
+      header={
```

Comment:
```suggestion
      headerComponent={
```
Since we're less scrupulous about proptypes, I wonder if it would be good to make the prop names more descriptive. I think this one popped out for me because the `header` prop here is a different thing than the `header` inside, and they might be confused?